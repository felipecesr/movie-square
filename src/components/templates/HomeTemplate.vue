<template>
  <div class="home">
    <page-header/>
    <movie-list/>
  </div>
</template>

<script>
import PageHeader from '@organisms/PageHeader.vue';
import MovieList from '@organisms/MovieList.vue';

export default {
  components: {
    PageHeader,
    MovieList
  }
}
</script>

<style lang="scss" scoped>
.home {
  display: grid;
  grid-column: 1 / -1;
  grid-gap: 30px;
  padding: 0 36px;
}
</style>
