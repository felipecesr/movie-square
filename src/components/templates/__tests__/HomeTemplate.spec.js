import { shallowMount } from '@vue/test-utils';
import HomeTemplate from '../HomeTemplate.vue';

describe('HomeTemplate.vue', () => {
  it('renders correctly', () => {
    expect(shallowMount(HomeTemplate).element).toMatchSnapshot();
  });
});
