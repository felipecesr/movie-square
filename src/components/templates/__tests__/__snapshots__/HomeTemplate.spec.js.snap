// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomeTemplate.vue renders correctly 1`] = `
<div
  class="home"
>
  <pageheader-stub />
   
  <movielist-stub />
</div>
`;
